app/uk/gov/hmrc/tariffclassificationfrontend/views/partials/activity_details_events_table.scala.html

```diff
@@ -40,10 +40,9 @@
                     event.details.`type` match {
                         case EventType.NOTE => {
                             val body = event.details.asInstanceOf[Note]
-                            progressive_disclosure(
+                            text_with_line_feeds(
```

Comment:
Can we call this something more intuitive, e.g. `paragraph`? or `text_block`?

Reply:
it's gone :)